# header
std_msgs/Header header

# cluster id
int16 id

# cluster attitude
# euler angles (roll, pitch, yaw)
geometry_msgs/Vector3 euler
# quaternion (x,y,z,w)
geometry_msgs/Quaternion  quat
